setup.py

```diff
@@ -5,12 +5,43 @@
 
 """
 
+import os
 import subprocess
 import distutils.cmd
+import setuptools.command.build_py
 from io import open
 from setuptools import setup
 
 
+class WebpackBuildCommand(setuptools.command.build_py.build_py):
+
+    """Prefix Python build with Webpack asset build"""
+
+    def run(self):
+        if not 'CI' in os.environ:
+            subprocess.run(['webpack', '--config', 'webpack.prod.js'], check=True)
```

Comment:
This does not work for me as webpack is not in my PATH. Perhaps you installed it as a global package?
It works with `npm run build` because npm adds `node_modules/.bin` to the PATH (https://docs.npmjs.com/cli/run-script)

```suggestion
            subprocess.run(['node_modules/.bin/webpack', '--config', 'webpack.prod.js'], check=True)
```

Reply:
This is probably because I have `node_modules/.bin` in my `PATH` actually. Using the local node_modules prefix seems most correct. I've always used the PATH pattern, but perhaps it's not standard.